packages/react-components/react-color-picker-preview/stories/src/ColorArea/ColorAreaDefault.stories.tsx

```diff
@@ -1,4 +1,52 @@
 import * as React from 'react';
+import { useId, Button, Label, makeStyles } from '@fluentui/react-components';
 import { ColorArea, ColorAreaProps } from '@fluentui/react-color-picker-preview';
+import { tinycolor } from '@ctrl/tinycolor';
 
-export const Default = (props: Partial<ColorAreaProps>) => <ColorArea {...props} />;
+const useStyles = makeStyles({
+  example: {
+    width: '300px',
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '10px',
+  },
+  previewColor: {
+    width: '50px',
+    height: '50px',
+    borderRadius: '4px',
+    border: '1px solid #ccc',
+  },
+});
+export const Default = () => {
+  const styles = useStyles();
+  const id = useId();
+  const hue = 0;
+  const [x, setX] = React.useState(100);
+  const [y, setY] = React.useState(100);
+  const [color, setColor] = React.useState('');
+  const onColorChange: ColorAreaProps['onColorChange'] = (_, data) => {
+    data.x && setX(data.x);
+    data.y && setY(data.y);
+  };
+
+  React.useEffect(() => {
```

Comment:
This effect is redundant, consider calling `setColor(tinycolor(_color).toHexString());` in `onColorChange` instead

Reply:
Is it fixed? The effect is still there 😄

Reply:
I reverted it accidentally 😄